lib/users-list.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
```

Comment:
More of a long-term thing:
Making the module stateful makes it harder to use parts of amphoraSearch independently or to use multiple ES hosts from the same app. I've only run into the first problem so far and it's not a big deal, but wanted to signal it here.